pkg/process/procutil/process_linux_test.go

```diff
@@ -275,6 +275,14 @@ func TestParseStatusLine(t *testing.T) {
 				ctxSwitches: &NumCtxSwitchesStat{},
 			},
 		},
+		{
+			line: []byte("NSpid:\t123\t456"),
+			expected: &statusInfo{
+				nspid:       456,
+				memInfo:     &MemoryInfoStat{},
+				ctxSwitches: &NumCtxSwitchesStat{},
+			},
+		},
```

Comment:
There were also tests that @mbotarro had added with respect to this in gopsutil - we should copy those tests.

Reply:
Good call, I'll port them over.

Reply:
This is added.